hugo/content/documentation/administration/acl/index.md

```diff
@@ -0,0 +1,250 @@
+
+If we define the second rule [on `ChanA1`] as `@~in` instead of `@in` then the inherited rule in `ChanA11` and `ChanA12` is interpreted in the context of `ChanA1`. That means that a user in `ChanA1` will be able to send text messages to `ChanA1`, `ChanA11`, and `ChanA12`. But a user not in `ChanA1` can not. Users in `ChanA11` and `ChanA12` will also not be able to send text messages to `ChanA1`, `ChanA11`, and `ChanA12`.
+
+#### `sub` Group
+
+The `sub` group applies to subchannels and has three optional parameters `a,b,c`. It is written as `@sub` or `@sub,a` or `@sub,a,b` or `@sub,a,b,c` where `a`, `b`, and `c` are integer numbers.
+
+| Parameter | Description                      | Default |
+| --------- | -------------------------------- | ------- |
+| a         | minimum number of common parents | `0`     |
+| b         | minimum path depth               | `1`     |
+| c         | maximum path depth               |         |
+
+Given the channel tree
+
+```text
+ChanA
+  ChanB
+    ChanC
+```
+
+| Selector     | On channel | Applies to user in        |
+| ------------ | ---------- | ------------------------- |
+| `@sub,0,0`   | `ChanA`    | `ChanA`, `ChanB`, `ChanC` |
+| `@sub,0,1`   | `ChanA`    |          `ChanB`, `ChanC` |
+| `@sub,0,2`   | `ChanA`    |                   `ChanC` |
+| `@sub,0,0,0` | `ChanA`    | `ChanA`                   |
+| `@sub,0,0,1` | `ChanA`    | `ChanA`, `ChanB`          |
+| `@sub,0,0,2` | `ChanA`    | `ChanA`, `ChanB`, `ChanC` |
+| `@sub,0,1,1` | `ChanA`    |          `ChanB`          |
+| `@sub,0,2,2` | `ChanA`    |                   `ChanC` |
+
+In other words, `b` and `c` define the child depth span (children of `b`-th to `c`-th depth).
+
+##### `sub` Example 1
+
+Given the channel tree
+
+```text
+ChanA
+  ChanB
+    ChanC
+```
+
+A `@sub` rule (equivalent to `@sub,0,1`) on `ChanA` applies to users in `ChanB` and `ChanC` but gives permission on `ChanA`.
+
+For example, if *Text message* is *denied* on `ChanA`, but a `@sub` rule on `ChanA` allows *Text message*, a user in `ChanB` or `ChanC` can send text messages to `ChanA`. A user in `ChanA` can not send a text message to `ChanA`.
+
+For example, with two rules on `ChanA`
+
+* `ChanA`: `@all` *Deny* *Text message*
+* `ChanA`: `@sub` *Allow* *Text message*
+
+=>
+
+* `ChanA`: user can not send text message to `ChanA`
+* `ChanB`: user can send text message to `ChanA`
+* `ChanC`: user can sent text message to `ChanA`
+
+### Evaluation Locality
+
+Given the channel tree
+
+```text
+ChanA
+  ChanB1
+  ChanB2
+```
+
+Defining an ACL rule `@in` on `ChanA` that inherits to subchannels means it *exists* in `ChanA` as well as its children `ChanB1` and `ChanB2`.
+But each rule applies to the channel *it is defined on*.

```

Comment:
This might be a bit misleading. I think a formulation like
```suggestion
Therefore, when checking this ACL inside e.g. `ChanB2` it acts as if it had been defined on it (aka: the evaluation of the ACL does not know that the original ACL was defined on `ChanA`).
```